snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -138,6 +139,33 @@ def _configure_container(self):
         subprocess.check_call([
             'lxc', 'config', 'set', self._container_name,
             'environment.LC_ALL', 'C.UTF-8'])
+        self._set_image_info_env_var()
+
+    def _set_image_info_env_var(self):
+        FAILURE_WARNING_FORMAT = (
+            'Failed to get container image info: {}\n'
+            'It will not be recorded in manifest.')
+        try:
+            image_info = json.loads(subprocess.check_output(
+                ['lxc', 'image', 'list',
+                 '--format=json', self._image]).decode())
```

Comment:
This looks a bit ugly:

    $ SNAPCRAFT_BUILD_INFO=1 snapcraft cleanbuild  
    Creating snapcraft-scintillantly-gobony-leena  
    Starting snapcraft-scintillantly-gobony-leena
    error: flag provided but not defined: --format
    Failed to get container image info: Command '['lxc', 'image', 'list', '--format=json', 'ubuntu:xenial/amd64']' returned non-zero exit status 2., output: b''
    It will not be recorded in manifest.
    Waiting for a network connection...
    Network connection established